### PR TITLE
Update transportation electrification section

### PR DESCRIPTION
**Pull request by rouille**
[Pull Request doc](https://breakthrough-energy.github.io/docs/user/git_guide.html#d-pull-request)

### Purpose
Update transportation electrification documentation

### What the code is doing
N/A

### Testing
Documentation has been generated locally and it looks as desired

### Where to look
2 figures has been updated, new cross-reference to sections have been created and paragraphs have been copied/pasted

### Usage Example/Visuals
N/A

### Time estimate
20min

### CODE REVIEW

File: docs/demand/transportation_electrification/methodology.rst [hunk 1]
```diff
@@ -36,8 +37,8 @@ The LDV and LDT vehicle miles traveled (VMT) by hour is presented in
 
    Light-duty vehicle/truck miles traveled by hour of the day for weekends and weekdays 
 
-Similarly, Figure :numref:`dwelling_over_24h` illustrates the percentage of vehicles
-parked and able to charge at each hour over a 24-hour period.
+Similarly, Figure :numref:`dwelling_over_24h` llustrates the percentage of vehicles
```
Comment (BainanXia) [inline docs/demand/transportation_electrification/methodology.rst]: `llustrates`->`illustrates`

File: docs/demand/transportation_electrification/methodology.rst [hunk 2]
```diff
@@ -486,30 +481,30 @@ row of the data table is a unique trip taken by the specified vehicle. 
...
-annual vehicle electricity demand. The base year and future projections of battery
-electric vehicle miles traveled are taken from NREL’s Electrification Futures Study
-:cite:p:`NREL_electric_technology_adoption`. BEV VMT is divided into VMT occurring in
-Urban Areas (UA) and Rural Areas (RA). UA is a term assigned by the U.S. Census Bureau
-and is described as areas with a population of 50,000 people or more
+The model is structured for a single base year (2017) and 33 future years (2018-2050).
+Each year is unique based on vehicle miles traveled (VMT) and fuel economy (miles per
+gallon of gasoline equivalent, mi/GGE), which together determine annual vehicle
+electricity demand. The base year and future projections of battery electric vehicle
+miles traveled are taken from NREL’s EFS:cite:p:`NREL_electric_technology_adoption`. BEV
```
Comment (BainanXia) [inline docs/demand/transportation_electrification/methodology.rst]: Do we want to add a space before `:cite:p:` here?

File: docs/demand/transportation_electrification/summary.rst [hunk 3]
```diff
@@ -1,31 +1,35 @@
...
+transportation electrification and its interactions with the electricity sector and
+other sectors such as building and manufacturing, a high resolution and accurate
+transportation electrification model integrated into the simulation framework with other
+sectors is necessary. A few key questions arise when determining the additional load
+from charging electric vehicles on the electric grid – how many electric vehicles of
+which types are expected to be on the roads by a given year and what are their typical
+trip patterns, which in turn determines when charging is most likely to happen.  As is
+frequently discussed, charging could have significant flexibility as to its precise
+timing, which ties this project into the Demand Flexibility project. In future,
+highly-integratedhighly integrated whole-economy energy systems, the optimal operation
```
Comment (BainanXia) [inline docs/demand/transportation_electrification/summary.rst]: `highly integrated` is duplicated.

File: docs/demand/transportation_electrification/summary.rst [hunk 4]
```diff
@@ -1,31 +1,35 @@
...
+vehicles (MDVs), and heavy-duty vehicles (HDVs), including fleet vehicles (e.g., buses).
 The core activities include (1) development of detailed data sets that represent the
 electricity demand patterns due to vehicle mobility, growth in electric vehicle
-penetration, and flexibility parameters for electrified light-duty vehicles (LDVs) and
-heavy-duty vehicles (HDVs) consisting of private and publicly-owned fleets; (2)
-development of code that can generate electricity demand profiles for LDVs and HDVs
-under different levels of charging intelligence, and (3) integration of the modeling
-capability for determining electrified LDV and HDV load profiles into the open-access,
-large-scale grid model developed by the Breakthrough Energy Grid Modeling team. 
+penetration, and flexibility parameters for electrified LDVs , LDTs, MDVs, and HDVs
```
Comment (BainanXia) [inline docs/demand/transportation_electrification/summary.rst]: Redundant space following `LDVs ,`

**Review by BainanXia — APPROVED** [review]
Thanks for taking care of this.

File: docs/demand/transportation_electrification/summary.rst (hunk 3 above)
Comment (rouille) [reply docs/demand/transportation_electrification/summary.rst]: Done

File: docs/demand/transportation_electrification/summary.rst (hunk 4 above)
Comment (rouille) [reply docs/demand/transportation_electrification/summary.rst]: Done

File: docs/demand/transportation_electrification/methodology.rst (hunk 2 above)
Comment (rouille) [reply docs/demand/transportation_electrification/methodology.rst]: Done

File: docs/demand/transportation_electrification/methodology.rst (hunk 1 above)
Comment (rouille) [reply docs/demand/transportation_electrification/methodology.rst]: Done

**Pull request merged**